packages/design-tokens/src/motion/motion.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I would keep duration and easing (maybe consider different name?) in separate files.
Any other thoughts?

Reply:
As duration and easing are relation to motion. I kept those in a same file named `motion`.